src/display/editor/outliner.js

```diff
@@ -260,13 +262,37 @@ class Outliner {
 }
 
 class Outline {
+  /**
+   * @returns {string} The SVG path of the outline.
+   */
   toSVGPath() {
     throw new Error("Abstract method `toSVGPath` must be implemented.");
   }
 
+  /**
+   * @returns {Object|null} The bounding box of the outline.
```

Comment:
That's not how existing getters are documented throughout the code-base, so please fix all occurrences in this patch.
```suggestion
  /**
   * @type {Object|null} The bounding box of the outline.
```